### DISCUSSION TITLE
Erro de validação PR

### DISCUSSION THREAD

**Discussion by Emeyas-Franco** [digitalinnovationone/dio-lab-open-source#48897] [Desafios DIO]
Alguém sabe me explicar como resolver o erro na imagem abaixo

![image](https://github.com/user-attachments/assets/450a3319-3b73-48e0-ba2a-4f4c2b2a8e6f)

**Comment by FernandaPortoAlegre** [comment]
No meu também está com o mesmo erro.

**Comment by elidianaandrade** [comment]
Olá, pessoal! Vi que, no caso do Emeyas, ele já conseguiu resolver, mas, para quem está enfrentando a mesma dificuldade, atente-se aos pontos de verificação que a automação retorna.
No caso do Emeyas, o nome do arquivo dele estava diferente do nome de usuário, pois estava com letras minúsculas, e a validação é case-sensitive.
Outro problema comum que pode ocorrer é a modificação de mais de um arquivo, quando apenas um deve ser alterado. Por isso, confiram sempre a aba "Files changed" para verificar as mudanças feitas.
